nonlingeo_precice.c

```diff
@@ -221,6 +221,7 @@ void nonlingeo_precice(double **cop, ITG *nk, ITG **konp, ITG **ipkonp, char **l
       .xboun = xboun,
       .ntmat_ = ntmat_,
       .vold = vold,
+	  .fn = fn,
```

Comment:
White spaces seem to have been mixed up a little. Please correct this (at several positions in the code).